styles/chess.com/catppuccin.user.less

```
@@ -56,6 +56,9 @@
     @accent: @catppuccin[@@flavor][@@accentColor];
 
     @highlight: @catppuccin[@@flavor][@@highlightColor];
+      
+    @light-cell: if(@flavor = latte, @catppuccin[@@flavor][@surface0], @catppuccin[@@flavor][@@accentColor]);
+    @dark-cell: if(@flavor = latte, @catppuccin[@@flavor][@@accentColor], @catppuccin[@@flavor][@surface0]);
```

```suggestion
    @light-cell: if(@flavor = latte, @surface0, @accent);
    @dark-cell: if(@flavor = latte, @accent, @surface0);
```
Is there a reason we aren't just doing this?

---

Oh... yeah. My bad. I'm not very experienced with CSS, but it should have been obvious to me that I didn't need to reuse the definition for those variables.